demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -439,19 +440,35 @@ def _collect_playbooks_dependencies(pack_playbooks: list, id_set: dict, verbose_
             ))
 
             # ---- incident fields packs ----
+            # playbook dependencies from incident fields should be marked as optional unless CommonTypes pack,
+            # as customers do not have to use the OOTB inputs.
             incident_fields = playbook_data.get('incident_fields', [])
             packs_found_from_incident_fields = PackDependencies._search_packs_by_items_names_or_ids(
                 incident_fields, id_set['IncidentFields'], exclude_ignored_dependencies)
             if packs_found_from_incident_fields:
-                pack_dependencies_data = PackDependencies._label_as_mandatory(packs_found_from_incident_fields)
+                common_types_pack_dependency = False
+                if COMMON_TYPES_PACK in packs_found_from_incident_fields:
+                    common_types_pack_dependency = True
```

Comment:
why do you need the flag? just update the mandatory dependency here.

also, this block causes lots of code duplication, please move to a function